doc/prometheus_collector.md

```diff
@@ -80,7 +80,7 @@ Example (simplified `prometheus_vm_memory_collector`):
 
 
 <pre><code>
-collect_mf_callback() = fun((<a href="prometheus_model.md#type-MetricFamily">prometheus_model:'MetricFamily'()</a>) -&gt; any())
+collect_mf_callback() = fun((<a href="http://www.erlang.org/edoc/doc/prometheus/doc/prometheus_model.html#type-MetricFamily">prometheus_model:'MetricFamily'()</a>) -&gt; any())
```

Comment:
Hmm, I checked the diff of the edown but these changes may not cause these changes.
https://github.com/uwiger/edown/compare/0.8.1...0.8.4

@deadtrickster So I've expected that you do `git restore` these wrong links on your local.